mos-platform/CMakeLists.txt

```diff
@@ -49,12 +49,12 @@ add_subdirectory(common)
 add_subdirectory(atari2600-common)
 add_subdirectory(atari2600-4k)
 add_subdirectory(atari2600-3e)
-add_subdirectory(atari5200-super)
+add_subdirectory(atari5200-supercart)
 add_subdirectory(atari8-common)
 add_subdirectory(atari8-dos)
 add_subdirectory(atari8-stdcart)
```

Comment:
Instead, it seems better to break out a term in the hierarchy: `atari8-cart-xegs`. This provide better parity with `atari8-dos`. It doesn't seem necessary to have the target dash hierarchy match the actual PARENT relation, so long as there isn't actually a `atari8-cart` target.